examples/README.markdown

```diff
@@ -0,0 +1,24 @@
+# Example Sinatra application
+
+This example illustrated how to configure and work towards Fishbrain identity
+services.
```

Comment:
This example illustrates how to configure and work towards Fishbrain identity services (staging environment).